src/sentry/migrations/0104_collect_relay_public_key_usage.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-09-17 11:47
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
```

Comment:
when redash is up again let's verify before merge that this table is basically empty (I think it is). otherwise this won't fly

Reply:
I would assume that we want both `CREATE INDEX CONCURRENTLY` and set this to false, as the comment indicates.

Reply:
let's wait for owners-migrations but I do think this is fine. `CREATE INDEX CONCURRENTLY` involves hand-written sql afaik and I don't think we need that stuff for an empty table...

Reply:
I took a look in prod and there are only a handful of rows in this table. You don't need to create the index concurrently with a low row count and low read/write traffic.